source/js/error-handling.js

```diff
@@ -75,4 +75,4 @@ var checkElements = function(arrayOfElements, callbackFunction) {
 module.exports = {
   checkElement  : checkElement,
   checkElements : checkElements
-};
+};
```

Comment:
Github still complains about this file not having a new line here.

Reply:
Github complaints?

Reply:
Yes, you added a new line to all other files, but not to this one.
![image](https://user-images.githubusercontent.com/45654887/60600230-13975300-9db0-11e9-89c3-0487fe43e98b.png)